### ISSUE TITLE
Accessibility Color Issues

### ISSUE THREAD

**Issue by wakingmorning** [badcamp/badcamp-2018#161]
Here are just a few quick accessibility issues that I noticed that could be addressed. Most revolve around color contrast and color changes on hover/focus.

## Potential Accessibility Color Contrast Issues
1. On the [Homepage](https://2018.badcamp.net/) the _Join the conversation at #Badcamp18 has a contrast ratio of 2.6 and should be about 3.0. Bumping up the font size may fix this, or darkening adjusting the background color. 
    <img width="1404" alt="screen shot 2018-08-21 at 1 47 55 pm" src="https://user-images.githubusercontent.com/6785398/44420591-5d869a00-a54c-11e8-8ffd-410bb722cdbf.png">
1. Webforms that have readonly fields have color contrast ratio issues. Not sure if any of these are public but it was picked up in a powermapper scan. 
    ```.webform-readonly input[type="date"],
    .webform-readonly input[type="datetime-local"],
    .webform-readonly input[type="email"],
    .webform-readonly input[type="number"],
    .webform-readonly input[type="password"],
    .webform-readonly input[type="search"],
    .webform-readonly input[type="tel"],
    .webform-readonly input[type="text"],
    .webform-readonly input[type="time"],
    .webform-readonly input[type="url"],
    .webform-readonly textarea {
      background: #ededed;
      border-color: #bbb;
      color: #717171;
    }
    ```
1. The `body` has a low contrast ratio of 1. Should be around 4.5 unless the font-size is greater then 18px. Currently the body looks to have a background of `#162b35`, with a color of `#012847`. Suggest making the color `#ffffff`? 
1. Alert buttons have a contrast ratio of 4.4 but should be 4.5 due to the small font size.
    ```.alert.action-button,
    .button.alert {
      background-color: #cc4b37;
      color: #fefefe;
    }
    .action-button[disabled].alert,.action-button[disabled].alert:focus,.action-button[disabled].alert:hover,.button.disabled.alert,.button.disabled.alert:focus,.button.disabled.alert:hover,.button[disabled].alert,.button[disabled].alert:focus,.button[disabled].alert:hover,.disabled.alert.action-button,.disabled.alert.action-button:focus,.disabled.alert.action-button:hover{background-color:#cc4b37;color:#fefefe}
    .badge.alert{background:#cc4b37;color:#fefefe}
    .button-group.alert .action-button,.button-group.alert .button{background-color:#cc4b37;color:#fefefe}
    .label.alert{background:#cc4b37;color:#fefefe}
    ```
    Bumping up the font size may resolve this.
1. The color contrast on this is approx. 1.4 and should be 4.5. If you bump the font-size to above 18px, you could get away with a contrast ratio of 3.0.
    ```.field--type-entity-reference .schedule fieldset[data-drupal-selector=edit-date] div.form-item legend,
    .field--type-entity-reference fieldset .schedule div[data-drupal-selector=edit-type] div.form-item legend,
    .field--type-file.form-wrapper .schedule div[data-drupal-selector=edit-type] div.form-item summary,
    .field--type-file.form-wrapper .schedule fieldset[data-drupal-selector=edit-date] div.form-item summary,
    .schedule .field--type-entity-reference fieldset[data-drupal-selector=edit-date] div.form-item legend,
    .schedule div[data-drupal-selector=edit-type] div.form-item .field--type-entity-reference fieldset legend,
    .schedule div[data-drupal-selector=edit-type] div.form-item .field--type-file.form-wrapper summary,
    .schedule div[data-drupal-selector=edit-type] div.form-item label,
    .schedule div[data-drupal-selector=edit-type] div.form-item table thead th h4,
    .schedule fieldset[data-drupal-selector=edit-date] div.form-item .field--type-file.form-wrapper summary,
    .schedule fieldset[data-drupal-selector=edit-date] div.form-item label,
    .schedule fieldset[data-drupal-selector=edit-date] div.form-item table thead th h4{
      background-color:#d3d3d3;
      color:#fff;
      font-weight:700;
      font-size:1.25rem;
      text-align:center;
      margin:0;
      padding:.5rem 1rem;
      min-width:100px
    }
    ````
1. The contrast ratio on the below zone is 2.6 and needs to be 4.5. Using a font-size of 15px, may resolve the issue. 
    ```#zone-highlighted {
      background-color:#c2993b;
      color:#fefefe;
    }
    ```

## Hover/focus colors
1. The main menu has no hover state and a focus state that may make the menu item hard to read (_text turns black and blends in with the background_).
1. The site name has no hover state and a focus state that may make the site name hard to read (_text turns black and blends in with the background_).
1. The site logo has no hover state. Being an SVG you could change the `fill` color on hover and on focus.
1. The mobile hamburger has no focus or active state to indicate that the menu is open. 
1. Input fields like the subscribe section have a very difficult to see focus state. 
    <img width="636" alt="screen shot 2018-08-21 at 3 04 19 pm" src="https://user-images.githubusercontent.com/6785398/44423229-98400080-a553-11e8-9395-3f2e8b5d780b.png">
    Maybe change the bottom border color on focus and maybe even on hover?
1. The sponsor logos should have a hover/focus state
    <img width="1145" alt="screen shot 2018-08-21 at 3 06 41 pm" src="https://user-images.githubusercontent.com/6785398/44423333-e0f7b980-a553-11e8-8e14-0514487b3bd4.png">
    Maybe an opacity change?
1. Social Icons need a hover/focus change
    <img width="416" alt="screen shot 2018-08-21 at 3 07 43 pm" src="https://user-images.githubusercontent.com/6785398/44423380-0389d280-a554-11e8-8b3e-81040f2cde6c.png">
    Maybe the same styling as the button hover?
1. The _built by_ links need hover/focus states. Even removing the underline may help.
    <img width="497" alt="screen shot 2018-08-21 at 3 11 36 pm" src="https://user-images.githubusercontent.com/6785398/44423617-975b9e80-a554-11e8-8fe4-f70eb8d9b676.png">
